documentation/getting_started/README.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started
+
+Welcome to WoTT - the first step to securing your IoT devices.
+
+## A brief overview of WoTT's services
+
+WoTT's primary goal is to simplify and manage end-to-end security for IoT developers. 
+We provide this in a number of ways:
+
+- Simplified encryption of device communication
```

Comment:
This is just minor stylistic preference, but let's use `*` for bullet points.

Reply:
Fixed

Reply:
Probably failed to commit that part as it still shows `-`.